src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -2068,6 +2068,26 @@ pread64(int fd, void *buf, size_t size, off_t offset) __attribute__((alias("prea
 ssize_t
 __pread64(int fd, void *buf, size_t size, off_t offset) __attribute__((alias("pread")));
 
+ssize_t
+__pread64_chk(int fd, void *buf, size_t size, off_t offset, size_t buflen)
+{
+	if (size > buflen) {
+		D_FATAL("buffer overflow detected in __pread64_chk().\n");
+		abort();
+	}
+	return pread(fd, buf, size, offset);
+}
+
+ssize_t
+__read_chk (int fd, void *buf, size_t size, size_t buflen)
```

Comment:
Whitespace issue.  Other than this line the patch is clang-format clean.
```suggestion
__read_chk(int fd, void *buf, size_t size, size_t buflen)
```

Reply:
Thank you! Will fix it in next commit.